README.md

```diff
@@ -1 +1,22 @@
-# hw01
+# Homework 1: Markdown Biography
+
+## Introduction
+
+![headshot](https://media-exp1.licdn.com/dms/image/C4E03AQE5isHwmTn2mw/profile-displayphoto-shrink_200_200/0?e=1605139200&v=beta&t=h21NrPkRVwGyF5AosXS3Fn6asKrGUhBjrKMWZEv0mr0)
```

Comment:
Great use of the picture function. I was unaware of how to do this, but seeing your work helped me out a lot. so Thanks :)

Reply:
Looks like you included most of the elements suggested for this homework, maybe next time you can try including a link as well!